src/Umbraco.Web.BackOffice/Mapping/MemberMapDefinition.cs

```diff
@@ -100,5 +103,18 @@ private static void Map(IMember source, ContentPropertyCollectionDto target, Map
         {
             target.Properties = context.MapEnumerable<IProperty, ContentPropertyDto>(source.Properties);
         }
+
+        /// <summary>
+        /// Maps an identity role to a member group display
+        /// </summary>
+        /// <param name="arg1"></param>
+        /// <param name="arg2"></param>
+        /// <param name="arg3"></param>
+        private void Map(IdentityRole source, MemberGroupDisplay target, MapperContext context)
+        {
+            //TODO: this is all that is mapped at this time, we're losing a lot of properties
```

Comment:
As per above comment re mapping - what do we need to return in the controller cases? This might be unwanted in the `MemberGroupController` cases.

Reply:
I don't think we need this map since we shouldn't be using the role manager for our member editors.